micro-transactions/src/test/java/com/aol/micro/server/transactions/TransactionFlowTest.java

```diff
@@ -38,7 +38,7 @@ public void flatMapTest(){
 		String result = TransactionFlow.of(transactionTemplate, this::load)
 										.flatMap(this::newTransaction)
 										.execute(10)
-										.get();
+										.orElse("");
 		
```

Comment:
Is this test not missing an assert ?

Reply:
Yes.. In my defence, I don't think it had one before :)